pkg/app/destroy_test.go

```diff
@@ -75,7 +86,7 @@ func TestDestroy(t *testing.T) {
 		wantDeletes := tc.deleted
 
 		var helm = &exectest.Helm{
-			Helm3:                tc.helm3,
+			Helm3:                true,
```

Comment:
why not remove Helm3 field?

Reply:
Because there is a helm version judgment
```
if !helm.IsHelm3() {
panic("helmfile has deprecated helm2 since v1.0")
}
```